app/allocations/middleware.test.js

```diff
@@ -82,21 +83,21 @@ describe('#setPagination', function() {
     it('creats correctly todayUrl', function() {
       expect(res.locals.pagination.todayUrl).to.exist
       expect(res.locals.pagination.todayUrl).to.equal(
-        '/allocations/week/2020-04-13/'
+        '/allocations/week/2020-04-13/outgoing/'
```

Comment:
Should our URLs have a trailing slash? I think they all actually get redirected to remove the trailing slash so maybe better to build with that in mind.

Reply:
As you say, the trailing slash should be indifferent (interestingly, in hapi it's an option and the default is not to redirect). I have removed them.